From: Norbert Preining <norbert@preining.info>
Date: Mon, 9 Oct 2017 21:09:18 +0900
Subject: no --detach in .desktop files (Closes: #877774)

Forwarded: not-needed
Closes: #877774

do not use --detach in the .desktop files, otherwise
mimetype handling of temporary files is broken as
the tmp files disappear too quickly. See #877774
---
 src/calibre/linux.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/calibre/linux.py b/src/calibre/linux.py
index c16d974..9941032 100644
--- a/src/calibre/linux.py
+++ b/src/calibre/linux.py
@@ -1097,7 +1097,7 @@ Name=E-book viewer
 GenericName=Viewer for E-books
 Comment=Viewer for E-books in all the major formats
 TryExec=ebook-viewer
-Exec=ebook-viewer --detach %f
+Exec=ebook-viewer %f
 Icon=calibre-viewer
 Categories=Graphics;Viewer;
 '''
@@ -1110,7 +1110,7 @@ Name=E-book editor
 GenericName=Editor for E-books
 Comment=Edit E-books in various formats
 TryExec=ebook-edit
-Exec=ebook-edit --detach %f
+Exec=ebook-edit %f
 Icon=calibre-ebook-edit
 Categories=Office;
 '''
@@ -1123,7 +1123,7 @@ Name=calibre
 GenericName=E-book library management
 Comment=E-book library management: Convert, view, share, catalogue all your e-books
 TryExec=calibre
-Exec=calibre --detach %F
+Exec=calibre %F
 Icon=calibre-gui
 Categories=Office;
 X-GNOME-UsesNotifications=true
